### PR TITLE
New version: SecombaGmbH.Boxcryptor version 2.55.2774

### PR DESCRIPTION
**Pull request by FlorianObermayer**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/88075)

### CODE REVIEW

**Comment by FlorianObermayer** [comment]
@microsoft-github-policy-service agree [company="Secomba GmbH"]

**Comment by FlorianObermayer** [comment]
@microsoft-github-policy-service agree company="Secomba GmbH"

File: manifests/s/SecombaGmbH/Boxcryptor/2.55.2774/SecombaGmbH.Boxcryptor.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: wix
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: REBOOT=REALLYSUPPRESS ARPSYSTEMCOMPONENT=1 /quiet /qn
```
Comment (stephengillie) [inline manifests/s/SecombaGmbH/Boxcryptor/2.55.2774/SecombaGmbH.Boxcryptor.installer.yaml:15]:
```suggestion
  Silent: /quiet
```

File: manifests/s/SecombaGmbH/Boxcryptor/2.55.2774/SecombaGmbH.Boxcryptor.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  SilentWithProgress: REBOOT=REALLYSUPPRESS ARPSYSTEMCOMPONENT=1 /qn
```
Comment (stephengillie) [inline manifests/s/SecombaGmbH/Boxcryptor/2.55.2774/SecombaGmbH.Boxcryptor.installer.yaml:16]:
```suggestion
  SilentWithProgress: /passive
```

**Comment by stephengillie** [comment]
@FlorianObermayer After installation, this package requires Microsoft WebView2 Runtime to run. Please include this as a dependency.

File: manifests/s/SecombaGmbH/Boxcryptor/2.55.2774/SecombaGmbH.Boxcryptor.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+- 3010
+UpgradeBehavior: install
+FileExtensions:
+- bc
+- bch
+- bckey
+- bclink
+Dependencies:
+  PackageDependencies:
+    - PackageIdentifier: Microsoft.EdgeWebView2Runtimes
```
Comment (Trenly) [inline manifests/s/SecombaGmbH/Boxcryptor/2.55.2774/SecombaGmbH.Boxcryptor.installer.yaml:28]:
```suggestion
    - PackageIdentifier: Microsoft.EdgeWebView2Runtime
```

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/s/SecombaGmbH/Boxcryptor/2.55.2774/SecombaGmbH.Boxcryptor.installer.yaml (hunk 3 above)
Comment (FlorianObermayer) [reply manifests/s/SecombaGmbH/Boxcryptor/2.55.2774/SecombaGmbH.Boxcryptor.installer.yaml]: Sorry for that! I happened to copy/paste the wrong lines when testing the validation system.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**